changelog/README.md

```diff
@@ -0,0 +1,67 @@
+Pending changelog
+=================
+
+This will get copied to dlang.org and cleared when master gets
+merged into stable prior to 2.072
+
+Please put your changelog entries into this folder.
+If you fix a bug fix, a changelog entry will automatically be generated.
+
+You can preview the changes with:
+
+```
+./build_changelog.d > doc.dd
+dmd -D doc.dd
+```
+
+Short changelog
+---------------
+
+Most users just want to quickly scan the changelog for areas of their interest,
+hence it's nice to provide a concise one-sentence information about your
+modification, it could look like this:
+
+```
+$(XREF range, padLeft) and $(XREF range, padRight) were added.
```

Comment:
When there's a `.long` file, the content of the `.short` file is going to be turned into a link/anchor, right? The `.short` must not contain links such as `XREF` then. Need to explain that here.